mindsdb/__main__.py

```diff
@@ -149,6 +144,16 @@ def close_api_gracefully(p_arr):
             print(traceback.format_exc())
             raise
 
+    os.environ['MINDSDB_STORAGE_PATH'] = config.paths['predictors']
+    if args.verbose is True:
+        os.environ['DEFAULT_LOG_LEVEL'] = 'INFO'
+        os.environ['LIGHTWOOD_LOG_LEVEL'] = 'INFO'
+    else:
+        os.environ['DEFAULT_LOG_LEVEL'] = 'ERROR'
+        os.environ['LIGHTWOOD_LOG_LEVEL'] = 'ERROR'
+        print("\nNOTE: Verbose is OFF [see --help for options]")
+    print('\n' + '='*7 + '\n') # print divider
```

Comment:
can you move these back to their original position ? If not, why did you move these down ?